### ISSUE TITLE
filter_duplicate parameter

### ISSUE THREAD

**Issue by SnowAndIceCode** [SeanLee97/AnglE#99]
Will changing the filter_duplicate parameter in the AngleDataCollator class to False affect the correctness of the subsequent three loss calculation results?

----------------------------
class AngleDataCollator:
    """
    AngleDataCollator. It will be implicitly used in AnglE.fit().
    It can only handle the tokenized data using AngleDataTokenizer.

    :param tokenizer:  PreTrainedTokenizerBase
    :param padding:   Union[bool, str, PaddingStrategy], padding strategy
    :param max_length:  Optional[int], max length
    :param return_tensors:  str
    :param filter_duplicate: bool. Whether filter duplicate data
    :param coword_random_mask_rate: float. Default 0.0.
        If set it greater than 0, the random maked token prediction will be added to the training loss.
    """

**Comment by SeanLee97** [comment]
Thank you for your suggestions. We will make it configurable in the next version.

**Comment by SeanLee97** [comment]
hi @SnowAndIceCode , the `filter_duplicate` now can be configurable in angle-trainer via `--filter_duplicate 0/1`. To use this feature, you should upgrade the `angle-emb` to the latest version (0.5.1) via `pip install -U angle-emb`.

Related PR: https://github.com/SeanLee97/AnglE/pull/100
